libazureinit/src/media.rs

```diff
@@ -66,7 +67,47 @@ fn default_preprov_type() -> String {
     "None".to_owned()
 }
 
-pub fn make_temp_directory() -> Result<(), Box<dyn std::error::Error>> {
+pub fn mount_media() -> Result<(), Error> {
+    let mount_status = Command::new("mount")
+        .arg("-o")
+        .arg("ro")
+        .arg("/dev/sr0")
+        .arg("/run/azure-init/tmp/")
+        .status()?;
+
+    if !mount_status.success() {
+        return Err(Error::SubprocessFailed {
+            command: "mount".to_string(),
+            status: mount_status,
+        });
+    }
+
+    Ok(())
+}
+
+pub fn remove_media() -> Result<(), Error> {
+    let umount_status = Command::new("umount")
+        .arg("/run/azure-init/tmp/")
```

Comment:
```suggestion
        .arg("/run/azure-init/media/")
```

Reply:
Fixed, and created a new const `PATH_MOUNT_MEDIA`, and replaced hard-coded ones with the new const.